dat/missions/dvaered/dvaered-colonel-transport.lua

```diff
@@ -0,0 +1,128 @@
+--[[
+<?xml version='1.0' encoding='utf8'?>
+<mission name="Dvaered Colonel Escort">
+ <unique />
+ <priority>4</priority>
+ <chance>86</chance>
+ <location>Bar</location>
+ <faction>Dvaered</faction>
+</mission>
+--]]
+--[[
+
+   Mission: Escort a Dvaered colonel
+   Description: Small mission where you escort a Dvaered Arsenal.
```

Comment:
I've seen other files that list the author before giving a description, but the mission template didn't do that.
Should I be doing that/are there rules about it that I haven't learned yet?

Reply:
There isn't really a convention there. I don't really think the author is necessary given it is a collective work, and all the changes are stored in git anyway.